percy-csharp-selenium/Percy.cs

```diff
@@ -172,82 +218,83 @@ public void Snapshot(String name, List<int> widths, int minHeight, bool enableJa
              * @param enableJavaScript Enable JavaScript in the Percy rendering environment
              * @param percyCSS Percy specific CSS that is only applied in Percy's browsers
         */
-        private void PostSnapshot(String domSnapshot, String name, List<int> widths, int minHeight, String url, bool enableJavaScript, String percyCSS)
+        private  void PostSnapshot(String domSnapshot, String name, List<int> widths, int minHeight, String url, bool enableJavaScript, String percyCSS)
         {
-            if (percyIsRunning == false)
+            if (!isPercyEnabled)
             {
                 return;
             }
 
-            // Build a JSON object to POST back to the agent node process
-            JsonObject json = new JsonObject();
-            json.Add("url", url);
-            json.Add("name", name);
-            json.Add("percyCSS", percyCSS);
-            json.Add("minHeight", minHeight);
-            json.Add("domSnapshot", domSnapshot);
-            json.Add("clientInfo", env.getClientInfo());
-            //json.Add("clientInfo", "percy-java-selenium/unknown");
-            json.Add("enableJavaScript", enableJavaScript);
-            json.Add("environmentInfo", env.getEnvironmentInfo());
-            //json.Add("environmentInfo", "selenium-java; MAC; chrome/85.0.4183.121");
-            // Sending an empty array of widths to agent breaks asset discovery
-            if (widths != null && widths.Count != 0)
+            if (widths == null)
+            {
+                widths =  new List<int>{};
+            }
+
+            if (percyCSS == null)
             {
-                json.Add("widths", JsonArray.Parse(GetSnapshotWidths(widths)));
+                percyCSS = "";
             }
 
-            string base_url = "http://localhost:5338/percy/snapshot";
+            var  param_dict = new Dictionary<string, object>(){
```

Comment:
query:
does the dictionary instance do the job of using the ContentType = JSON

just wanted to understand which part handles 
```
        StringEntity entity = new StringEntity(json.toString(), ContentType.APPLICATION_JSON);
```